Resources/Prototypes/SimpleStation14/Entities/Clothing/Head/hardsuit-helmets.yml

```
@@ -13,3 +13,27 @@
     highPressureMultiplier: 0.3
     lowPressureMultiplier: 1000
 
+- type: entity
+  parent: ClothingHeadHardsuitWithLightBase
+  id: ClothingHeadHelmetHardsuitHoP
+  noSpawn: true
+  name: head of personnel's plated hardsuit helmet
+  description: Fancy helmet for a fancy hardsuit. Doesn't offer much damage protection.
+  components:
+  - type: Sprite
+    sprite: SimpleStation14/Clothing/Head/Hardsuits/hopsuit.rsi
+  - type: Clothing
+    sprite: SimpleStation14/Clothing/Head/Hardsuits/hopsuit.rsi
+  - type: PointLight
+    color: "#ff2034"
+  - type: PressureProtection
+    highPressureMultiplier: 0.525
+    lowPressureMultiplier: 10000
```

```suggestion
    lowPressureMultiplier: 1000
```

Hardsuit helmet values are _all_ over the place, but 1000 seems to be normal for non-combat ones.

---

holy fuck they are. I have no clue what really is a standard